ports/esp32/boards/UM_FEATHERS3NEO/board.json

```diff
@@ -0,0 +1,26 @@
+{
+    "deploy": [
+        "deploy.md"
+    ],
+    "docs": "",
+    "features": [
+        "BLE",
+        "Battery Charging",
+        "External Flash",
+        "External RAM",
+        "Feather",
+        "JST-SH",
+        "RGB LED",
+        "USB-C",
+        "WiFi"
+    ],
+    "features_non_filterable": [],
+    "images": [
+        "unexpectedmaker_feathers3_neo.jpg"
+    ],
+    "mcu": "esp32s3",
+    "product": "FeatherS3 Neo",
+    "thumbnail": "",
+    "url": "https://feathers3.io",
```

Comment:
This redirects to https://esp32s3.com/feathers3.html.  I think it should instead be https://esp32s3.com/feathers3neo.html